filo-scala/src/main/scala/org.velvia.filo/RowReader.scala

```diff
@@ -186,7 +191,7 @@ object RowReader {
     final def getField(reader: RowReader, columnNo: Int): String = reader.getString(columnNo)
     override final def getFieldOrDefault(reader: RowReader, columnNo: Int): String = {
       val str = reader.getString(columnNo)
-      if (str == null) DefaultString else str
+      if (str == "") DefaultString else str
```

Comment:
No, we don't want to switch this from null.  We really do want to check for null, and we can't use Option because it would be too slow here.